### PR TITLE
Support swift tune for diffusion models

### PR DESCRIPTION
**Pull request by XDUWQ**
@tastelikefeet @wenmengzhou @wangxingjun778 
* test input:
image
* swift output:
image
image

* origin output:
image
image

### CODE REVIEW

File: facechain/merge_lora.py [hunk 1]
```diff
@@ -4,13 +4,22 @@
...
     LORA_PREFIX_TEXT_ENCODER = "lora_te"
     if from_safetensor:
         state_dict = load_file(lora_path, device=device)
+    elif os.path.exists(os.path.join(lora_path, 'unet')):
+        if not is_swift_available():
+                    raise ValueError(
+                        'Please install swift by `pip install ms-swift` to use efficient_tuners.'
+                    )
+        from swift import Swift
+        Swift.from_pretrained(pipeline.unet, os.path.join(lora_path, 'unet'))
```
Comment (tastelikefeet) [inline facechain/merge_lora.py:21]: pipeline.unet=Swift.from_pretrained(pipeline.unet, os.path.join(lora_path, 'unet'))
这样是不是比较好写

File: facechain/merge_lora.py [hunk 2]
```diff
@@ -4,13 +4,22 @@
...
 from safetensors.torch import load_file
+from modelscope.utils.import_utils import is_swift_available
 
 
 def merge_lora(pipeline, lora_path, multiplier, from_safetensor=False, device='cpu', dtype=torch.float32):
     LORA_PREFIX_UNET = "lora_unet"
     LORA_PREFIX_TEXT_ENCODER = "lora_te"
     if from_safetensor:
         state_dict = load_file(lora_path, device=device)
+    elif os.path.exists(os.path.join(lora_path, 'unet')):
```
Comment (tastelikefeet) [inline facechain/merge_lora.py:15]: unet特异性不太高，用swift当文件夹会不会好些

Comment (XDUWQ) [reply facechain/merge_lora.py]: 已改正

File: facechain/merge_lora.py (hunk 1 above)
Comment (XDUWQ) [reply facechain/merge_lora.py]: 已改正

**Review by tastelikefeet — APPROVED** [review]

**Pull request merged**